SMLHelper/Handlers/ItemActionHandler.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace SMLHelper.V2.Handlers
+{
+    using SMLHelper.V2.Patchers;
+    using System;
+
+    /// <summary>
+    /// A handler class for registering your custom middle click actions for items
+    /// </summary>
+    public class ItemActionHandler
+    {
+        /// <summary>
+        /// Registers a middle click action for an <see cref="InventoryItem"/> based off of its <see cref="TechType"/>
+        /// </summary>
+        /// <param name="targetTechType">The <see cref="TechType"/> which the middle click action will be assigned to</param>
+        /// <param name="callback">The method which is called when the <see cref="InventoryItem"/> was middle-clicked</param>
+        /// <param name="tooltip">The secondary tooltip which appears in the description of the item</param>
+        public static void RegisterMiddleClickAction(TechType targetTechType, Action<InventoryItem> callback, string tooltip = null)
+        {
+            ItemActionPatcher.CustomItemActions.Add(targetTechType, callback);
+            if (!string.IsNullOrEmpty(tooltip)) ItemActionPatcher.CustomItemActionTooltips.Add(targetTechType, tooltip);
```

Comment:
The tooltip value should be routed through the LanguagePatcher, if possible, so it can over handled with the language overrides.

Reply:
Recommendation:  
Use `LanguageHandler.SetLanguageLine($"MCAction_{targetTechType}", tooltip);` to pass the tooltip to the Language Helper

Then, use `(Language.main.Get($"MCAction_{targetTechType}")` to retrieve the patched language line in the Harmony patch method.